testing/unittests/espapicmdtests.cpp

```diff
@@ -312,7 +312,14 @@ void sortAndWriteBack(string& input)
     }
 }
 
-Init setup;
+Init& ensureSetup()
```

Comment:
I think this should have been rejected as a false positive
* The file hooks are defined in a dll this is dependent on, so the initiasation order is guaranteed.
* This code is not dependent on any hooks being installed.

However there are a few issues with this code.
i) setup should be a member of EspApiCmdTest.  That should also fix the dependency issue, and also ensure files are deleted at a better time.
ii) Init is not a good name for a class - dangerous for poluting the global namespace.  Either a nested class or include the EspApCmdTestInit

i) setup should be a static